profiles/arch/amd64/use.mask

```diff
@@ -4,6 +4,10 @@
 # Unmask the flag which corresponds to ARCH.
 -amd64
 
+# Andrew Ammerlaan <andrewammerlaan@gentoo.org> (2023-07-11)
+# Mask secureboot flag on arches where sbsigntools is not keyworded
+-secureboot
```

Comment:
I'd change the messages on arches where it's being unmasked, it sounds confusing when it says Mask but then it's being unmasked.